src/LuauExt.cpp

```diff
@@ -51,11 +51,10 @@ std::optional<nlohmann::json> parseDefinitionsFileMetadata(const std::string& de
     return std::nullopt;
 }
 
-Luau::LoadDefinitionFileResult registerDefinitions(
```

Comment:
We still need the `typeCheckForAutocomplete` here for the old solver right?

Reply:
See PR description:
```
In addition, loadDefinitionFile and by extension registerBuiltinGlobals, though still taking a typeCheckForAutocomplete argument, that argument no longer does anything within Luau internals, meaning the default to false is safe and it can be removed from explicit use.
```
This includes the old solver.

Reply:
Ah fair enough, didn't check Luau internals but indeed you are right